spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -455,23 +455,23 @@ of the spec as laid out in this document.
 === LRA Client API
 
 For completeness the proposal supports clients that wish to directly
-control LRAs and participants. To support this class of user an instance of
+control LRAs and participants. To support this, the class of user can obtain an instance of
```

Comment:
The original English was correct and the replacement you propose is not correct English.

If the original was not comprehensible then please say which bit doesn't make sense an I can change it.

Reply:
I agree with @mmusgrov I think it reads better as it was before actually

Reply:
I will revert this after feedback on the the next comment

Reply:
or if it is the word *class" that is the problem how about replacing with the equivalent "type" or "kind"  or "To support such users and instance of ... can be ...",  etc